apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1961,6 +1965,368 @@ static void libc_stdio_setvbuf_tc(void)
+	TC_ASSERT_EQ("lib_memsistream", memsistream.buffer, (FAR char *)(buf));
+
+	lib_memsistream((FAR struct lib_memsistream_s *)&memsistream, buf + 2, BUFLEN);
+	TC_ASSERT_EQ("lib_memsistream", (memsistream.buffer - (FAR char *)(buf)), 2);
+	TC_ASSERT_EQ("lib_memsistream", memsistream.buflen, BUFLEN);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase         libc_stdio_memsostream_tc
+* @brief            Initializes a stream for use with a fixed-size memory buffer
+* @scenario         Initializes a stream for use with a fixed-size memory buffer
+* @apicovered       lib_memsostream
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_memsostream_tc(void)
+{
+	FAR char buf[BUFLEN];
+
+	struct lib_memsostream_s memsostream;
+
+	lib_memsostream((FAR struct lib_memsostream_s *)&memsostream, buf, BUFLEN);
+	TC_ASSERT_EQ("lib_memsostream", memsostream.buffer, (FAR char *)(buf));
+
+	lib_memsostream((FAR struct lib_memsostream_s *)&memsostream, buf + 4, BUFLEN);
+	TC_ASSERT_EQ("lib_memsostream", (memsostream.buffer - (FAR char *)(buf)), 4);
+	TC_ASSERT_EQ("lib_memsostream", memsostream.buflen, (BUFLEN - 1));	/* Save space for null terminator, hence checing with (BUFLEN-1)*/
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase         libc_stdio_nullinstream_tc
+* @brief            Initializes a NULL stream. The initialized stream will return only EOF
+* @scenario         Initializes a NULL stream. The initialized stream will return only EOF
+* @apicovered       lib_nullinstream
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_nullinstream_tc(void)
+{
+	FAR struct lib_instream_s *stream;
+	struct lib_instream_s nullinstream;
+
+	lib_nullinstream(&nullinstream);
+	stream = &nullinstream;
+	TC_ASSERT_EQ("lib_nullinstream", nullinstream.nget, 0);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase         libc_stdio_nulloutstream_tc
+* @brief            Initializes a NULL stream. The initialized stream will write all data to the bit-bucket
+* @scenario         Initializes a NULL stream. The initialized stream will write all data to the bit-bucket
+* @apicovered       lib_nulloutstream
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_nulloutstream_tc(void)
+{
+	FAR struct lib_outstream_s *stream;
+	struct lib_outstream_s nulloutstream;
+
+	lib_nulloutstream(&nulloutstream);
+	stream = &nulloutstream;
+	TC_ASSERT_EQ("lib_nulloutstream", nulloutstream.nput, 0);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase         libc_stdio_rawinstream_tc
+* @brief            Initializes a stream for use with a file descriptor
+* @scenario         Initializes a stream for use with a file descriptor
+* @apicovered       lib_rawinstream
+* @precondition     NA
+* @postcondition    NA
+*/
+static void libc_stdio_rawinstream_tc(void)
+{
+	int fd;
+	char *filename = VFS_FILE_PATH;
+
+	struct lib_rawinstream_s rawinstream;
+
+	fd = open(filename, O_RDONLY);
+	TC_ASSERT_NEQ("open", fd, OK);
```

Comment:
This is wrong because you should check whether fd is not greater than equal to 0.
So It should be TC_ASSERT_GEQ("open", fd, 0);